splice_machine/src/test/java/com/splicemachine/derby/utils/StatisticsAdminIT.java

```diff
@@ -1242,6 +1267,19 @@ public void testCollectStatsWithMoreSplits() throws Exception {
         methodWatcher4.execute("drop table TAB_WITH_MORE_SPLITS");
     }
 
+    @Test
+    public void testNoAssertionFailureInSketch() throws Exception {
```

Comment:
This test does not test anything anymore, since assertions are disabled, right?
We should either remove it, @Ignore it with a comment, or enable assertions I suppose

Reply:
That's right. I enabled the assertion in the latest commit.